### PR TITLE
feat: populate geom field on planter_registrations

### PR DESCRIPTION
**Pull request by luacmartins**

### CODE REVIEW

**Comment by luacmartins** [comment]
@arunbakt this should populate the `geom` field in the `planter_registrations` table

File: src/data.js [hunk 1]
```diff
@@ -57,114 +54,139 @@ class Data {
...
         const insert = {
           text: `INSERT INTO tree_attributes
           (tree_id, key, value)
           values
           ($1, $2, $3)
           RETURNING *`,
-          values: [id, attribute.key, attribute.value]
+          values: [id, attribute.key, attribute.value],
         }
-        const stored = await this.pool.query(insert);
```
Comment (arunbakt) [inline src/data.js:89]: Isn't semicolons recommended even though it is optional?

**Comment by arunbakt** [comment]
Hi @luacmartins , Can you give me some context surrounding this change?  Since there are lots of formatting related changes, I can 't see the actual change in the PR.  Also, I saw you have opted to not have semicolons when formatting the code. Isn't the recommended thing is to have semicolons even though they are optional in javascript?

File: src/data.js (hunk 1 above)
Comment (luacmartins) [reply src/data.js:89]: You are absolutely correct! I just realized that this project did not have a Prettier config file. I made changes to add the config file and include semicolons. Thanks!

**Comment by luacmartins** [comment]
@arunbakt The changes I made are in `createPlanterRegistration`. These changes should add the `geom` column to the `planter_registrations` table. They were required by the admin panel to get the country of each planter. Let me know if that's still not clear. Cheers!

**Review by arunbakt — APPROVED** [review]
Looks good to me, it was difficult to isolate the actual change amidst the formatting changes. It would help if it were a two step PR first one with just formatting changes followed by the actual logic change.

**Comment by nmcharlton** [comment]
@arunbakt @luacmartins Can we try to get this PR fixed and merged?

**Comment by luacmartins** [comment]
I fixed the conflicts. Should be good to be merged.

**Comment by nmcharlton** [comment]
@arunbakt @ZavenArra This is ready to be merged by someone with the authority.

**Pull request merged**